src/Navbar/Navbar.vue

```diff
@@ -8,23 +8,25 @@
                tabindex="0"
                v-for="item in items"
                :key="item"
-               @focus="itemSelect($event, 'menu', item)"
-               @blur="itemDeselect($event, 'menu')"
-               @mouseover="itemSelect($event, 'menu', item)"
-               @mouseleave="itemDeselect($event, 'menu')">
+               @focus="itemActivate($event, item)"
+               @blur="itemDeselect({delay: 250})"
+               @touchstart="itemClick($event, item)"
+               @mouseover="itemActivate($event, item)"
+               @mouseleave="itemDeselect({delay: 250})">
             {{item}}
           </div>
         </div>
         <div class="cta"></div>
       </div>
       <transition name="dropdown">
-        <div tabindex="0"
-             @focus="itemSelect($event, 'dropdown', itemSelected)"
-             @blur="itemDeselect($event, 'dropdown')"
-             @mouseover="itemSelect($event, 'dropdown', itemSelected)"
-             @mouseleave="itemDeselect($event, 'dropdown')"
-             class="dropdown"
-             v-if="!!dropdown.visible">
+        <div class="dropdown"
+             :tabindex="dropdown.tabindex"
```

Comment:
Inner links will be navigable with arrows (when we make them so). Dropdown is focusable, because when you tab away from a menu item, it hides content for this item, which is what we want. If we click within the dropdown, however, you effectively blur from the menu item and unless the dropdown is focusable, it will close the dropdown. I can maintain the state of being focused on the dropdown (as opposed to being focused on something else) in JS, but that doesn't seeem right.

Reply:
Okay, looking at Stripe's behavior (which perhaps we shouldn't take as perfect), there is no focus possible on the dropdown or the inner links. Therefore it is only possible to either directly click an item in the dropdown, or highlight it via keyboard navigation. **However**, it seems the keyboard navigation (up/down) is **only** possible when the dropdown is **opened** with keyboard navigation (tabbing).
I _think_ that might be the best behavior, since up and down keys scroll the window by default - we should only override that behavior if necessary and expected.
I understand your dilemma though. Perhaps we want inner links to be focusable (e.g. mousedown -> mouseleave -> mouseup) but not via keyboard navigation? This way you can keep the dropdown menu open on blur **if** one of its children holds focus. My main point is that it's not of any use to the user to focus the dropdown container itself, functionally speaking, since it doesn't have an action, it doesn't scroll, it doesn't _do_ anything.